.github/workflows/dokka-github-pages.yml

```diff
@@ -7,6 +7,8 @@ on:
       - release/*
     tags:
       - "[0-9]+.[0-9]+.[0-9]+"
```

Comment:
I wonder if this is still necessary.

Reply:
Yep we can remove that if the release trigger works as expected. I think it's a bug on github side, I've found other project having issues with `ref_name` being unset or wrong.

I also found this issue https://github.com/actions/runner/issues/2788 which seems to affect the release trigger as well :/